trino-aws-proxy-glue/src/main/java/io/trino/aws/proxy/glue/rest/GlueDeserializer.java

```diff
@@ -70,9 +71,18 @@ public T deserialize(JsonParser parser, DeserializationContext context)
                 if (type == null) {
                     return (T) context.handleUnexpectedToken(context.getContextualType(), parser);
                 }
+
                 if (type == SdkBytes.class) {
                     sdkField.set(builder, SdkBytes.fromByteArray(fieldValue.binaryValue()));
                 }
+                else if (type == StringColumnStatisticsData.class) {
+                    System.out.println("$$$");
```

Comment:
I removed this `if` clause and the tests still pass. Please update the tests so that they fail without this change.